### ISSUE TITLE
How to generate inline html ?

### ISSUE THREAD

**Issue by BeepBeepBopBop** [xoofx/markdig#820]
Hello, I am new to Markdig and I have the following use-case:

I am trying to write a UI similar to Chat GPT's, with a string of character followed by a circle. Here is the simplified code:

```
<p style="display: inline;">
     <span style="display: inline;">
         @((MarkupString)Markdig.Markdown.ToHtml(MessageViewModel.Text))
     </span>

     <span style="display: inline;">
         X
     </span>
 </p>
```

**-> The issue that I have is that Markdown generates block elements. Is there a way to configure so that the "X" in the code above appear on the same line as the html generated by Markdig ?**

Thanks a lot for your help.

**Comment by AmadeusW** [comment]
I don't think it's a bug on markdig. You can achieve what you want in CSS, using `display: flex` rather than `display:inline`, with with `align-items: flex-end`. Here's details: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

**Comment by xoofx** [comment]
Agreed, not really a bug or an issue with markdig, but HTML/CSS related.
